rllib/algorithms/algorithm_config.py

```diff
@@ -2116,6 +2118,11 @@ def training(
                 should set this setting to False.
                 Note that this setting is only relevant if the new API stack is used
                 (including the new EnvRunner classes).
+            learner_config_dict: A dict to insert any settings accessible from within
```

Comment:
Although I am voting for simplicity, I do not like this many roads lead to Rome approach. Overiding the `AlgorithmConfig`  is not too sophisticated imo and the user is now confrotned with the question what to use when. I consider a single truth approach as less confusing even though here a bit more effort is needed. But this is my taste.

Reply:
Maybe you are right. It definitely adds another option to get tangled up in. ...
Let me think about this more ...